docs/operate/get-started/other-hardware/manage-modules.md

```diff
@@ -62,21 +62,21 @@ Use [GitHub Actions](https://docs.github.com/actions) to automatically build and
 
    ```yaml
    on:
-     push:
-       tags:
-         - "[0-9]+.[0-9]+.[0-9]+"
+   push:
```

Comment:
This indentation is different from what's here https://github.com/viamrobotics/build-action. is this correct?

Reply:
YAML is like Python, it doesn't care as long as you're consistent. But I can update this to follow that 4-space convention! Sorry about that.